src/terminal/node/terminal-backend-module.ts

```diff
@@ -22,13 +22,20 @@ export default new ContainerModule(bind => {
 @injectable()
 class TerminalExpressContribution implements BackendApplicationContribution {
     private terminals: Map<number, any> = new Map()
-    private logs: string[] = []
+
+    getShellExecutablePath() {
+        if (process.platform === 'win32') {
```

Comment:
Here you could use the `isWindows` constant from the `os` module and then we do not have to repeat the `win32` string.